### PR TITLE
feat: add Arbitrum Sepolia support

### LINKED ISSUE #51: Add support for Arbitrum Sepolia

### PR DESCRIPTION
**Pull request by douglance**
closes #51

### CODE REVIEW

File: src/utils/configBuilders.ts [hunk 1]
```diff
@@ -44,12 +45,16 @@ export function buildRollupConfigData({
...
   rollupContracts: RollupContracts;
   validators: Wallet[];
   batchPoster: Wallet;
+  parentChainId: number;
 }): RollupConfigData {
+  const parentChainRpcUrl = getRpcUrl(parentChainId);
+
   return {
     'chain': {
       'info-json': [
```
Comment (gzeoneth) [inline src/utils/configBuilders.ts:60]: https://github.com/OffchainLabs/orbit-deployment-ui/blob/450aed9f88443320f3cc6c548d743159830107cd/src/utils/configBuilders.ts#L63
also need to change this

Comment (spsjvc) [reply src/utils/configBuilders.ts:60]: Fixed in https://github.com/OffchainLabs/orbit-deployment-ui/pull/52/commits/e9744e50e9fe667c400becbab6a4664fbb22ab13

File: src/utils/deployRollup.ts
Comment (gzeoneth) [inline src/utils/deployRollup.ts:1]: https://github.com/OffchainLabs/orbit-deployment-ui/blob/e9744e50e9fe667c400becbab6a4664fbb22ab13/src/utils/configBuilders.ts#L220-L230

we need the parentChainId here too

Comment (gzeoneth) [reply src/utils/deployRollup.ts:1]: also `parent-chain-node-url`

**Comment by TucksonDev** [comment]
For some reason, even though the Batch poster field is set (the default value though), the node config doesn't seem to have its private key set:

```
"batch-poster": {
      "max-size": 90000,
      "enable": true,
      "parent-chain-wallet": {
        "private-key": ""
      }
    },
```

Tested in 2 browsers (in case it was a cache issue)

File: src/utils/deployRollup.ts
Comment (spsjvc) [reply src/utils/deployRollup.ts:1]: Fixed in https://github.com/OffchainLabs/orbit-deployment-ui/pull/52/commits/1cde91f73b9b02685f6a9f9e102dbc070739f76f

**Pull request merged**